junit-platform-suite-commons/src/main/java/org/junit/platform/suite/commons/SuiteLauncherDiscoveryRequestBuilder.java

```diff
@@ -206,6 +212,11 @@ private List<ClassSelector> selectClasses(Class<?>... classes) {
 		return AdditionalDiscoverySelectors.selectClasses(classes);
 	}
 
+	private MethodSelector selectMethod(Class<?> methodClass, String methodName, String methodParameterTypes) {
+		selectedClassNames.add(methodClass.getName());
```

Comment:
I made a class called SelectMethodsSuite
```
@Suite
@SelectMethod(clazz = MultipleTestsTestCase, name = test)
public class SelectMethodsSuite {
}
```
I thought it was necessary  because modifying the class  'SelectClassesSuite' will effect the test `SuiteEngineTests.selectClasses()` then I added this test to  `SuiteEngineTests`
```
	@Test
	void selectMethod() {
		// @formatter:off
		EngineTestKit.engine(ENGINE_ID)
				.selectors(
						selectMethod(MultipleTestsTestCase.class, "test")
				)
				.execute()
				.testEvents()
				.assertThatEvents()
				.haveExactly(1, event(test(SelectMethodSuite.class.getName() + ".singleTestMethod"), finishedSuccessfully()));
		// @formatter:on
	}
```
I'm not sure if this is the test needed test case, which is why I did not push the changes yet. 

I reflect on the fact that one must write tests for both the "old" and the "new" Juint5 code °^° 

@mpkorstanje